src/framework-configuration.js

```diff
@@ -1,7 +1,24 @@
 import * as TheLogManager from 'aurelia-logging';
-import {ViewEngine} from 'aurelia-templating';
+import { camelCase, ValueConverterResource, BindingBehaviorResource } from 'aurelia-binding';
+import {ViewEngine, ViewResources, HtmlBehaviorResource, _hyphenate} from 'aurelia-templating';
 import {join} from 'aurelia-path';
 import {Container} from 'aurelia-dependency-injection';
+import { metadata } from 'aurelia-metadata';
+
+
+/** @typedef ConfigInfo
```

Comment:
I haven't seen this notation before. How is this different from declaring an interface?

Reply:
I think interface won't work in the mixes environment so I used jsdoc instead

Reply:
the jsdoc notation is similar to interface, except that it could work without any issues in a JS environment. Once we switch to TS, it can simply be changed to interface